helpers/release/main.go

```diff
@@ -113,72 +114,76 @@ func main() {
 		panic(err)
 	}
 	fmt.Println("✅ Wrote VERSION")
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(sleep)
 	// - update version.go
 	if err := writeVersionGo(nextVer); err != nil {
 		panic(err)
 	}
 	fmt.Println("✅ Wrote version.go")
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(sleep)
 	// - update CHANGELOG.md
 	if err := writeChangelog(gitVer, nextVer); err != nil {
 		panic(err)
 	}
 	fmt.Println("✅ Updated CHANGELOG.md")
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(sleep)
 
 	// - create PR
 	//   git checkout -b release/vX.Y.Z
 	if err := gitCoRel(nextVer); err != nil {
 		panic(err)
 	}
 	fmt.Printf("✅ Created branch release/v%s\n", nextVer.String())
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(sleep)
 
 	// commit changes
 	if err := gitCommit(nextVer); err != nil {
 		panic(err)
 	}
 	fmt.Printf("✅ Committed changes to release/v%s\n", nextVer.String())
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(sleep)
 
-	fmt.Println("🏁 Done")
-	fmt.Printf("⚠ Prepared release of gopass %s. Now push this branch to your fork and open a pull request against gopasspw/gopass master.\n", nextVer.String())
-	fmt.Println()
-	fmt.Printf("⚠ After that PR has been merged tag it as 'v%s' and push that tag to kick of the release proecess.\n", nextVer.String())
+	fmt.Println("🏁 Preparation finished")
+	time.Sleep(sleep)
+
+	fmt.Printf("⚠ Prepared release of gopass %s.\n", nextVer.String())
+	time.Sleep(sleep)
+
+	fmt.Printf("⚠ Push this branch to your fork and open a PR against gopasspw/gopass master.\n")
```

Comment:
This could include the `git` command to do so since we already created the branch above, so we know its name.